### ISSUE TITLE
failure installing help pages 4.5.7

### ISSUE THREAD

**Issue by prometheanfire** [wwmm/pulseeffects#468]
http://dpaste.com/1WR6C4X has the log

downstream bug is at https://bugs.gentoo.org/681184

https://bugs.gentoo.org/681164 seems like the source of the issue

**Comment by prometheanfire** [comment]
tested using the old `help/pt_BR/pt_BR.po` file (4.5.6) and it worked, formatting error?

**Comment by wwmm** [comment]
Errors in the translation files can break the compilation. But I just tested a clean compilation on Arch Linux and everything is fine. So this does not seem to be the issue.

**Comment by wwmm** [comment]
Unless some problem happened when the github tarball was generated. I am testing with the master branch

**Comment by wwmm** [comment]
I downloaded `4.5.7` tarball and at least on Arch it compiles and install with `ninja install` with no problems.

**Comment by wwmm** [comment]
If I am not mistaken meson uses `itstool` and `Yelp` in the install phase to build translation. Could it be that there is a problem with these packages in Gentoo?

**Comment by wwmm** [comment]
In case it helps here in Arch I have:
`meson 0.49.2`
`itstool 2.0.2+5+g676f3f7`
`yelp 3.32.0`

**Comment by prometheanfire** [comment]
meson 0.48.2
itstool 2.0.2
yelp-xsl 3.20.1
yelp-tools 3.18.0

**Comment by prometheanfire** [comment]
also on 

```
meson 0.49.2
itstool 2.0.2
yelp-tools 3.28.0
yelp-xsl 3.30.1
```

newest versions available here

**Comment by wwmm** [comment]
Strange. Although I have the same warning in my PC
```
Warning: Could not merge pt-BR translation for msgid:
<em style="strong">Feed-forward</em> - The sidechain input is connected to compressor's input. More aggressive compression.
Warning: Could not merge pt-BR translation for msgid:
<em style="strong">Feed-back</em> - The sidechain input is connected to compressor's output. Vintage-style more accurate compression.
Warning: Could not merge pt-BR translation for msgid:
<em style="strong">APO</em> - Digital biquad filters derived from canonic analog biquad prototypes digitalized through Bilinear transform. These are <link href="https://shepazu.github.io/Audio-EQ-Cookbook/audio-eq-cookbook.html" its:translate="yes"> textbook filters </link> which are implemented as in the <_:link-1/> software.
```
Somehow it install. In any case I will try to fix it. I imagine once this warning goes away it will build on Gentoo too

**Comment by prometheanfire** [comment]
thanks for looking at it

**Comment by wwmm** [comment]
I released a new version. After removing a few spaces and changing how a link was inserted the warnings went away. But as the installation was already happening here I don't know if your problem was fixed too. Try to test `4.5.8`

**Comment by prometheanfire** [comment]
worked for me, thanks

fixed in `4.5.8`
